src/enclave/rpc_sessions.h

```diff
@@ -29,8 +29,8 @@ namespace ccf
 
   static constexpr size_t max_open_sessions_soft_default = 1000;
   static constexpr size_t max_open_sessions_hard_default = 1010;
-  static constexpr ccf::Endorsement endorsement_default =
-    ccf::Endorsement{ccf::Authority::SERVICE};
+  static const ccf::Endorsement endorsement_default = {
+    ccf::Authority::SERVICE, std::nullopt};
```

Comment:
What's the nullopt for here?

Reply:
The `acme_configuration`, which is optional, i.e. only present when the auhority is `ACME`.